.github/workflows/needs_release_notes.yml

```diff
@@ -11,7 +11,12 @@ jobs:
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@8558fd74291d67161a8a78ce36a881fa63b766a9  # v5.0.0
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: Run Pull Request Labeler
+      uses: actions/labeler@v5.0.0
```

Comment:
Could we switch back to using the SHA instead? It will be updated find by dependabot, but it's generally recommended to use the SHA instead of a tag for security reasons.